Adyen/Analytics/AnalyticsProvider/TelemetryTracker.swift

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// : nodoc:
+public enum TelemetryFlavor {
+    case components(type: String)
```

Comment:
Looks like this string is the payment method type. We have a new enum for this, maybe we can use it here too?

Reply:
Yes you're right! It was build right before Mo merged the changes, I'll change it to the enum type in the following PR. Nice catch! Thanks for remembering me